tests/basic/basic_test.go

```diff
@@ -194,6 +194,44 @@ var _ = Describe("AppTasksDown", func() {
 	})
 })
 
+// This test scales up and down an application and checks if app has actually scaled accordingly
+var _ = Describe("AppScaleUpAndDown", func() {
+	It("has to scale up and scale down the app", func() {
+		var err error
+		var contexts []*scheduler.Context
+		for i := 0; i < Inst().ScaleFactor; i++ {
+			contexts = append(contexts, ScheduleAndValidate(fmt.Sprintf("appscaleupdown-%d", i))...)
+		}
+
+		Step("increase scale factor of all apps by 1", func() {
```

Comment:
change this comment to just say "scale up the application" or something similar instead of saying by 1
same for scale down